frontend/src/stores/FilterQueryStore.ts

```diff
@@ -34,12 +34,19 @@ export default class FilterQueryStore extends SimpleStore<FilterQuery> {
   /**
    * Updates the state of the filterQuery by synchronizing it with the {@link FilterStateStore}.
    */
-  update(): void {
+  public update(): void {
     const filter = this.filterStateStore.getValue();
     const filterQuery = this.convertToFilterQuery(filter);
     this.mergeState(filterQuery);
   }
 
+  /**
+   * Resets the store to its initial state.
+   */
+  public reset(): void {
```

Comment:
As this is part of a lot of stores now, maybe we can move this to the common base type with `this.setState(this.getInitialValue());` as the default implementation.